<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bmilaskuri</class>
 <widget class="QMainWindow" name="Bmilaskuri">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Painoindeksilaskin</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="text_gender">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sukupuoli</string>
    </property>
   </widget>
   <widget class="QLabel" name="arvo_gender">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_paino">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Paino (kg)</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_pituus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pituus (cm)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_paino">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="suffix">
     <string> kg</string>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_pituus">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="suffix">
     <string> cm</string>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
   </widget>
   <widget class="QSlider" name="slider_paino">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSlider" name="slider_pituus">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="text_indeksi">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Painoindeksi</string>
    </property>
   </widget>
   <widget class="QLabel" name="arvo_indeksi">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_tulkinta">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tulkinta</string>
    </property>
   </widget>
   <widget class="QLabel" name="arvo_tulkinta">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QMenu" name="menuToimenpide">
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Toimenpide</string>
    </property>
    <addaction name="actionAlkutila"/>
    <addaction name="actionLopeta"/>
   </widget>
   <widget class="QMenu" name="menuSukupuoli">
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="title">
     <string>Sukupuoli</string>
    </property>
    <addaction name="actionNainen"/>
    <addaction name="actionMies"/>
   </widget>
   <addaction name="menuToimenpide"/>
   <addaction name="menuSukupuoli"/>
  </widget>
  <action name="actionAlkutila">
   <property name="text">
    <string>Alkutila</string>
   </property>
  </action>
  <action name="actionLopeta">
   <property name="text">
    <string>Lopeta</string>
   </property>
  </action>
  <action name="actionNainen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Nainen</string>
   </property>
  </action>
  <action name="actionMies">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mies</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>spin_paino</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_paino</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_paino</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_paino</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_pituus</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_pituus</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_pituus</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_pituus</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
